.rustfmt.toml

```diff
@@ -4,7 +4,7 @@ tab_spaces = 4
 newline_style = "Auto"
 use_small_heuristics = "Default"
 indent_style = "Block"
-wrap_comments = false
+wrap_comments = true # changed
 format_code_in_doc_comments = false
 comment_width = 80
```

Comment:
I'd personally prefer 90, but as long as we're consistent it's fine

Reply:
I'm fine with `90`, changed to that